xmipp

```diff
@@ -294,12 +295,16 @@ def runSelectedMode(parser: ErrorHandlerArgumentParser, args: argparse.Namespace
 	else:
 		# If method was none of the above, exit with error
 		logger(red(f"Mode \"{args.mode}\" not recognized. {COMMON_USAGE_HELP_MESSAGE}"), forceConsoleOutput=True)
-		exitXmipp(retCode=1, sendAPI=False)
+		exitXmipp(retCode=1)
 	
+	# Send API message
+	if sendAPI and os.path.exists(VERSION_FILE):
+		sendApiPOST()
+
```

Comment:
Try avoiding code duplication. This check, has been duplicated to send the API message before the success final message, but this should not have been needed.

Until I refactor the code to make it cleaner, it should do though.